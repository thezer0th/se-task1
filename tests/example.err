Error in line 3: 3 6:00 Winnicka 5:00 Banacha 6:00
Error in line 8: Niedobry 4.333 39
