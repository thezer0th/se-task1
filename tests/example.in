1 6:00 Banacha 7:00 Pasteura 8:00 Winnicka
2 6:00 Winnicka 7:00 Banacha 8:00 Pasteura
3 6:00 Winnicka 5:00 Banacha 6:00
4 8:00 Winnicka 9:00 Banacha
Niedlugi 2.30 3
Dlugi Drogi 3.80 60
Dlugi Tani 3.70 60
Niedobry 4.333 39
? Banacha 1 Winnicka
? Banacha 1 Winnicka 4 Banacha
