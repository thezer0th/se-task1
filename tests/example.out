! Dlugi Tani; Dlugi Tani; Niedlugi
:-|
3
